asyncscope.md

```diff
@@ -1825,7 +1825,32 @@ spec here
 
 ## `execution::nest()`
 
-spec here
+::: add
+__§34.9.11.14 `std::execution::nest` [exec.nest]__
```

Comment:
should this be 34.9.11.15 to make room for async_scope_token? or maybe this should go before async_scope_token to keep the numbers in the same order

Reply:
I'm not confident this is in the right place, but I picked it as the next sender adaptor after [`stopped_as_error`](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p2300r9.html#spec-execution.senders.adapt.stopped_as_error). I think that, since `async_scope_token` is a concept and not a sender adaptor, it probably belongs in another section, but I'm not sure which one.

Reply:
I think this is the right place for `nest` ... it is a sender adaptor, after all. I think the async_scope types and concepts could go under [exec.utils]. All of this stuff will need forward declarations in [exec.syn].